apps/system/js/sound_manager.js

```diff
@@ -248,6 +298,9 @@
         self.CEAccumulatorTime = value;
       }
     });
+
+    Service.registerState('isHeadsetConnected', this);
```

Comment:
I do share the concerns about `Service` getting messy really quickly.

But, interestingly, I think `isHeadsetConnected` is a valid use case :)
It is a piece of state global to the phone that is of interest for multiple modules.

That said, I'll go nit pick on other `Services` uses now :)